src/styles/card.css

```
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  box-sizing: border-box;
+}
+
+.card__img > img {
```

Add this class to img tag
```suggestion
.card__img {
```